### ISSUE TITLE
Generate default frontend_addon_name from title

### ISSUE THREAD

**Issue by Fosten** [plone/cookieplone-templates#110]
“volto-addon” is provided as the default frontend_addon_name

https://github.com/plone/cookieplone-templates/blob/e0d452f23e1dcfde1cbcb216c9e58360a5f8e6d9/frontend_addon/cookiecutter.json#L3

Whereas it could be generated programmatically like so 

```
"frontend_addon_name": "{{ cookiecutter.title | slugify }}",
```

Similar to how it is done in a cookieplone project

https://github.com/plone/cookieplone-templates/blob/e0d452f23e1dcfde1cbcb216c9e58360a5f8e6d9/project/cookiecutter.json#L4

**Comment by davisagli** [comment]
@Fosten Good catch, this is currently handled correctly when the frontend_addon template is used as part of a project, but not when it is used as a standalone template. Would you be willing to submit a PR with your proposed change?
